cfgov/v1/jinja2/v1/includes/molecules/info-unit-2.html

```diff
@@ -122,7 +124,11 @@
                             href="{{ link.url }}"
                             {%- if link.aria_label -%}
                                 aria-label="{{ link.aria_label }}"
-                            {%- endif %}>{{ link_text }}</a>
+                            {%- endif %}>
+                            {% if link.make_link_boldface == True%}<strong>{% endif %}
```

Comment:
All of these

`{% if link.make_link_boldface == True %}`

can be simplified to

`{% if link.make_link_boldface %}`

for brevity's sake.

Reply:
Resolved in https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/46dfb5ded5a17778287e9baa644e29e4f278e9a6